File: src/main/java/com/DavidM1A2/othertest/common/worldGeneration/structure/base/iterator/IChunkIterator.java
package com.DavidM1A2.afraidofthedark.common.worldGeneration.structure.base.iterator;

import net.minecraft.util.math.ChunkPos;

import java.util.List;

/**
 * Used to get a list of chunks to iterate over
 */
public interface IChunkIterator
{
    /**
     * @return A list of chunks to iterate over
     */
    List<ChunkPos> getChunks();
}

File: src/main/kotlin/com/davidm1a2/afraidofthedark/common/world/structure/old/base/iterator/IChunkIterator.kt
package com.davidm1a2.afraidofthedark.common.world.structure.old.base.iterator

import net.minecraft.util.math.ChunkPos

/**
 * Used to get a list of chunks to iterate over
 */
interface IChunkIterator {
    /**
     * @return A list of chunks to iterate over
     */
    fun getChunks(): List<ChunkPos>
}